client/k8s/application/util/K8sUtil.go

```diff
@@ -437,6 +452,39 @@ func (impl K8sUtil) DeleteJob(namespace string, name string, clusterConfig *Clus
 	return nil
 }
 
+func (impl K8sUtil) GetK8sConfigAndClients(clusterConfig *ClusterConfig) (*rest.Config, *http.Client, *kubernetes.Clientset, error) {
+	restConfig, err := impl.GetRestConfigByCluster(clusterConfig)
+	if err != nil {
+		impl.logger.Errorw("error in getting rest config by cluster", "err", err, "clusterName", clusterConfig.ClusterName)
+		return nil, nil, nil, err
+	}
+	k8sHttpClient, err := OverrideK8sHttpClientWithTracer(restConfig)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	k8sClientSet, err := kubernetes.NewForConfigAndClient(restConfig, k8sHttpClient)
+	if err != nil {
+		impl.logger.Errorw("error in getting client set by rest config", "err", err, "restConfig", restConfig)
+		return nil, nil, nil, err
+	}
+	return restConfig, k8sHttpClient, k8sClientSet, nil
+}
+
+func (impl K8sUtil) DiscoveryClientGetLiveZCall(cluster *ClusterConfig) ([]byte, error) {
+	_, _, k8sClientSet, err := impl.GetK8sConfigAndClients(cluster)
+	if err != nil {
+		return nil, err
+	}
+	//using livez path as healthz path is deprecated
+	response, err := impl.GetLiveZCall(LiveZ, k8sClientSet)
+	return response, err
```

Comment:
we cannot be sure that this err will always be handled in the parent func, handle